### ISSUE TITLE
Judo integration

### ISSUE THREAD

**Issue by hayd** [MichaelHatherly/Docile.jl#156]
An alternative to mkdocs, damn it looks purrty...

https://github.com/dcjones/Judo.jl

(putting this in Docile rather than Lexicon considering the rewrite branch :) )

**Comment by MichaelHatherly** [comment]
Yeah, that would be nice as an option. Since we'll just be generating plain old markdown output in Docile presumably that can then just be run through Juno afterwards?
